ouroboros-consensus-cardano/changelog.d/20231026_110631_teodora.danciu_cardano_ledger_for_8_6.md

```diff
@@ -0,0 +1,30 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+
+### Non-Breaking
+
+ - Updated to newer `cardano-ledger-*` packages:
+    * `cardano-ledger-api-1.7`
+    * `cardano-ledger-core-1.8`
+    * `cardano-ledger-shelley-1.7`
+    * `cardano-ledger-babbage-1.5`
+    * `cardano-ledger-conway-1.10`
+    * `cardano-protocol-tpraos-1.0.3.6`
+- Updated to `typed-protocols-0.1.1`
```

Comment:
@amesgen  Added this changelog. 
I'm assuming that he changes needed to make things work with ledger don't need to be included here, since they don't seem to be touching "The surface", but I'm not sure... For example: not triggering the HF in Conway. Would you say this should be added here?

Reply:
> I'm assuming that he changes needed to make things work with ledger don't need to be included here, since they don't seem to be touching "The surface", but I'm not sure...

"The surface" currently still is everything (until we have to time do sth like #296), so in particular, this fragment should be "Breaking" instead of "Non-Breaking" as it changes the types of eg a few queries, which will require changes in `cardano-api`.

> For example: not triggering the HF in Conway.

Not mentioning this is fine I think, you currently can't HF out of Conway anyway as there is no successor era :smile:

Reply:
Makes sense, I forgot about the breaking change to the query, though I integrated it in cardano-api myself :facepalm: 

I updated now, moving the ledger updates to "breaking" and mentioning the change in the query type